docs/split-inputs.md

```diff
@@ -136,8 +136,22 @@ and split the fuzz input into several parts, the corpus elements will no longer
 be valid image files, and you won't be able to simply add image files to your
 corpus.
 
-[FuzzedDataProvider] is a class whose constructor accepts `const uint8_t*,
-size_t` arguments. Below is a quick overview of the available methods.
+### Main concepts
+
+* [FuzzedDataProvider] is a class whose constructor accepts `const uint8_t*,
+  size_t` arguments. Usually, you would call it in the beginning of your
+  `LLVMFuzzerTestOneInput` and pass `data, size` provided by the fuzzing engine.
+* Once an FDP object is constructed using the fuzz input, you can consume the
+  data from the input by calling the FDP methods listed below.
+* If there is not enough data left in the buffer passed to the constructor, FDP
```

Comment:
What do you mean by "not enough data"? Not enough for what?

Reply:
Not enough to fulfill the request, e.g. the user calls `ConsumeBytes(10)` while there are just 4 bytes left in the buffer, then only those 4 bytes will be returned. Tried to clarify, PTAL

Reply:
LGTM